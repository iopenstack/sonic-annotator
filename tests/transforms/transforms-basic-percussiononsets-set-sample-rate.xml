<transform
    id="vamp:vamp-example-plugins:percussiononsets:onsets"
    sampleRate="22050">
</transform>
