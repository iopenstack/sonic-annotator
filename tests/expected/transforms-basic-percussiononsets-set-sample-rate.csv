"./audio/3clicks8.wav",0.766258503
,1.648616780
,2.414875283
